src/index.ts

```diff
@@ -336,6 +342,13 @@ export async function convert(
   from?: string,
   to?: string
 ): Promise<void> {
-  const node = await read(inp, from)
-  await write(node, out, to)
+  // TODO: finish refactoring this so that it handles missing
+  // inp and out args (i.e. reading from stdin, writing to stdout)
+
+  const inpFile = vfile.create({ path: inp })
+  const inpCompiler = await match(inp, from)
+  const node = await inpCompiler.parse(inpFile)
```

Comment:
I you mentioned this briefly during our call, but wanted to make sure we don't forget to follow up.
This change is causing the contents to always be empty for me, no matter what format/conversion I tried.

Reply:
Thanks for the prod. This should now be fixed and there are additional test cases in place to catch any regressions.